src/Microsoft.ML.Data/Prediction/Calibrator.cs

```diff
@@ -1280,6 +1286,43 @@ internal static int GetBinIdx(float output, float min, float binSize, int numBin
             return binIdx;
         }
 
+        bool ISingleCanSaveOnnx.SaveAsOnnx(OnnxContext ctx, string[] outputNames, string featureColumn)
+        {
+            _host.CheckValue(ctx, nameof(ctx));
+            _host.CheckValue(outputNames, nameof(outputNames));
+            _host.Check(Utils.Size(outputNames) == 2);
+
+            const int minimumOpSetVersion = 9;
+            ctx.CheckOpSetVersion(minimumOpSetVersion, "NaiveCalibrator");
+
+            string opType = "Sub";
+            var minVar = ctx.AddInitializer(Min, "Min");
```

Comment:
**Aproval:** So in general your PR LGTM, and I'd approve it, but I think it's better that @harishsk and/or @Lynx1820 take a look on this.

There are some quirks with onnx variable types and the options for some onnx operators which have never been clear to me. This quirks usually don't appear with our tests but can appear when using other options, or when working with NimbusML. So maybe they'll have more comments here ❓

Reply:
@antoniovs1029 Sounds good, thank you! I will wait for @harishsk / @Lynx1820 's reviews, and then update the names of `ModelInput` and `ModelInputNonStandard` per your comment above.